src/app/components/header/header.component.html

```diff
@@ -100,21 +105,36 @@
                 <ng-container *ngIf="tokens.length > 2">
                     <div class="divider"></div>
                     <button
-                        mat-flat-button
+                        mat-icon-button
                         class="balances-button"
+                        matTooltip="{{
+                            tokenBalancesOpen
+                                ? ''
+                                : 'Show all on-chain balances'
+                        }}"
```

Comment:
Nitpick: Sorry I forgot again if this was easily possible in Angular. But this really bloats the template with details. If possible it would be cool if this could be defined as some kind of computed data and you just have something like `matTooltip="{{ showBalancesTooltip }}"`. What do you think?

Reply:
There's not a feature like the computed data of Vue. You can use a function but that would get invoked for every change detection. But I just found out that there is `matTooltipDisabled` property you can use to not show the tooltip conditionally. That makes more sense in this case. Thank you for helping me finding this!